packages/wallet/src/Wallets/BaseWallet.ts

```diff
@@ -482,11 +556,13 @@ export class PersonalWallet implements ObservableWallet {
           )
         });
 
-    this.publicStakeKeys$ = createPublicStakeKeysTracker({
-      addresses$: this.addresses$,
-      bip32Account: this.bip32Account,
-      rewardAccounts$: this.delegation.rewardAccounts$
-    });
+    this.publicStakeKeys$ = isBip32PublicCredentialsManager(this.#publicCredentialsManager)
+      ? createPublicStakeKeysTracker({
+          addresses$: this.addresses$,
+          bip32Account: this.#publicCredentialsManager.bip32Account,
+          rewardAccounts$: this.delegation.rewardAccounts$
+        })
+      : new TrackerSubject(of(new Array<PubStakeKeyAndStatus>()));
```

Comment:
This PubStakeKeyAndStatus observable seems to be a convenience utility for CIP-30 integration. It takes all the reward accounts with their delegation statuses and tries to find the PubKey (as opposed to the pub key hash encoded in the reward account). The reward account of the shared wallet is a script hash, and from this we cant try to find any meaningful equivalent. That endpoint of CIP-30 (getRegisteredPubStakeKeys) is not supported by this kind of wallets as there are no pub key reward accounts, CIP-30 would need a getRegisteredScriptsForStake or something, so we can have an equivalent

Reply:
Got it. In that case, I think we should replace it with something like `stakeCredentials$`, but since we're not using it for script wallets it's ok to skip it in this PR. Could you please create a ticket in Misc Technical Improvements and add a comment for this line?

Reply:
ticket created, see LW-9980